pkgs/by-name/br/bruno/package.nix

```
@@ -67,6 +71,10 @@ buildNpmPackage rec {
     # disable telemetry
     substituteInPlace packages/bruno-app/src/providers/App/index.js \
       --replace-fail "useTelemetry({ version });" ""
+
+    # fix version reported in sidebar and about page
+    ${jq}/bin/jq '.version |= "${version}"' packages/bruno-electron/package.json | sponge packages/bruno-electron/package.json
+    ${jq}/bin/jq '.version |= "${version}"' packages/bruno-app/package.json | sponge packages/bruno-app/package.json
```

```suggestion
    # fix version reported in sidebar and about page
    jq '.version |= "${version}"' packages/bruno-electron/package.json | sponge packages/bruno-electron/package.json
    jq '.version |= "${version}"' packages/bruno-app/package.json | sponge packages/bruno-app/package.json
```

`${jq}` comes from `hostPlatform` by default. Just use the one come with `nativeBuildInputs`.

---

Oh nice, thanks